src/main/java/duke/DeleteInput.java

```diff
@@ -0,0 +1,23 @@
+package duke;
+
+public class DeleteInput extends InputHandler {
+
+    public DeleteInput(Ui ui, TaskList taskList) {
+        super(ui, taskList);
+    }
+
+    @Override
+    public String handle(String input) throws EmptyDescriptionException {
+        if (input.length() == 6) {
+            throw new EmptyDescriptionException("error");
+        }
+
+        char taskIndex = input.charAt(7);
```

Comment:
i like this separation of lines. allows me to understand the flow of logic slightly better.